variables.tf

```
@@ -1,9 +1,60 @@
-variable "email_address" {
-  description = "Email address associated with the member account. Required input for the GuardDuty member invitation."
-  type        = string
+variable "enable" {
+  description = "(Optional) Enable GuardDuty monitoring and feedback reporting. Setting to false is equivalent to 'suspending'GuardDuty. Defaults to true."
+  type        = bool
+  default     = true
 }
 
-variable "guardduty_master_detector_id" {
-  description = "GuardDuty Detector ID for master account"
-  type        = string
+variable "filter" {
```

Since this is a list, make it plural
```suggestion
variable "filters" {
```

---

Updated